hudi-common/src/main/java/org/apache/hudi/common/util/FSUtils.java

```diff
@@ -47,12 +47,13 @@
 import java.util.Arrays;
 import java.util.LinkedList;
 import java.util.List;
+import java.util.Objects;
 import java.util.Map.Entry;
-import java.util.UUID;
 import java.util.function.Function;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Stream;
+import java.util.UUID;
```

Comment:
It seems the old position is correct? For the import order, the priority of the class name is larger than package name.

Reply:
ok - so where UUID be then?

Reply:
The original place before changing?